style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  background-color: black;
+  color: yellow;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+.main-container {
+  padding: 100px;
+  width: 540px;
+}
+
+.timer {
+  font-size: 3rem;
+  display: flex;
+  margin: 30px;
```

Comment:
create default margin variable. You already pass 30px to few classes so it could be a variable